features/step_definitions/page_steps.rb

```diff
@@ -163,77 +164,77 @@
 Then "I should be able to edit that page's content" do
-  sleep(1)
   within_frame 'mercury_iframe' do
-    within '[data-mercury]' do
-      page.body.should =~ /#{@content}/
+    within "section[data-mercury='full']" do
+      expect(page.body).to match(/#{@content}/)
     end
   end
 end
 
 Then "I should be redirected to the new page" do
-  current_path.should == '/editor/some-new-page'
+  expect(current_path).to eq("/editor/some-new-page")
 end
 
 Then "I should be redirected to the home page" do
-  current_path.should =~ /^\/editor\/?$/
+  expect(current_path).to match(/^\/editor\/?$/)
 end
 
 Then "I should be redirected to the about page" do
-  current_path.should == '/about'
+  expect(current_path).to eq("/about")
 end
 
-Then /I should see the (.+) in the sitemap/ do |page|
+Then (/I should see the (.+) in the sitemap/) do |page|
   within '.mercury-panel .sitemap' do
-    page.should have_content "#{page}"
+    expect(page).to have_content "#{page}"
   end
 end
 
 Then "I should see the path in the redirects" do
   within '.mercury-panel .redirects' do
-    page.should have_content '/redirect-path'
+    expect(page).to have_content "/redirect-path"
   end
 end
 
 Then "I should not see the path in the redirects" do
   within '.mercury-panel' do
-    page.should_not have_content '/redirect-path'
+    expect(page).to_not have_content "/redirect-path"
   end
 end
 
 Then "I should not see the duplicate path in the redirects" do
   within '.mercury-panel' do
-    page.should_not have_content '/about'
+    expect(page).to_not have_content "/about"
   end
 end
 
 Then "I should be alerted to the duplicate redirect" do
-  page.should have_content 'Source URL must be unique'
+  expect(page).to have_content 'Source URL must be unique'
 end
 
-Then /^I should see that seo info on the page/ do
+Then (/^I should see that seo info on the page/) do
   allow_hidden_elements do
 
     visit current_path.gsub('/editor', '')
     within('title') do
-      page.should have_content('This is a new title for the browser')
+      expect(page).to have_content('This is a new title for the browser')
     end
-    page.should have_css('meta[name="keywords"]')
-    page.should have_css('meta[content="some_keyword, someother_keyword"]')
-    page.should have_css('meta[name="description"]')
-    page.should have_css('meta[content="This is a description of the page"]')
+
+    expect(page).to have_css('meta[name="keywords"]')
+    expect(page).to have_css('meta[content="some_keyword, someother_keyword"]')
+    expect(page).to have_css('meta[name="description"]')
+    expect(page).to have_css('meta[content="This is a description of the page"]')
 
   end
 end
 
 Then "the current page should be publicly available" do
   step %(I visit the current page's public path)
-  current_path.should_not =~ /\/editor/
-  page.should have_content('This is a published page')
+  expect(current_path).to_not match(/\/editor/)
+  expect(page).to have_content('This is a published page')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.